reference/tools/user-guide.md

```diff
@@ -6,184 +6,270 @@ aliases: ['/docs/dev/how-to/migrate/from-mysql/','/docs/dev/how-to/migrate/incre
+- Use DM to migrate MySQL data to TiDB for both full and incremental data import
 
-### MySQL data migration
+#### Data replication between TiDB/MySQL clusters
 
-- Mydumper, Loader and Syncer -> DM:
+You can use TiDB Binlog to replicate data between TiDB clusters. You can also use TiDB Binlog to replicate data to the downstream MySQL cluster.
 
-    It is tedious to migrate MySQL data to TiDB using Mydumper, Loader, and Syncer. DM provides an integrated data migration approach that improves usability. DM can be also used to merge the sharded schemas and tables.
+#### Full backup and restore of the data in TiDB/MySQL clusters
 
-- Loader -> TiDB Lightning:
+- Use the Mydumper tool for full data backup 
+- Use the Lightning tool for full data restore
 
-    TiDB Lightning integrates the logical data import function of Loader. See [TiDB Lightning TiDB-backend document](/reference/tools/tidb-lightning/tidb-backend.md#migrating-from-loader-to-tidb-lightning-tidb-backend) for details. It is used to perform full data restoration.
+### For TiDB 3.1
 
-## Data migration solutions
+#### Migrating MySQL data to TiDB
 
-For TiDB 2.1, 3.0, and 3.1 versions, this section introduces data migration solutions in typical application scenarios.
+If the MySQL data volume is in TBs:
 
-### Full link data migration solutions for v3.0
+- Use Mydumper to export MySQL full data as a backup
+- Use Lightning to import the full MySQL backup data into TiDB cluster
+- Use DM to replicate incremental MySQL data to TiDB
 
-#### Migrating MySQL data to TiDB
+If the MySQL data volume is in GBs:
 
-If the volume is more than TBs of data, the recommended migration steps are:
+- Use DM to migrate MySQL data to TiDB for both full and incremental data import
 
-1. Export full MySQL data using Mydumper;
-2. Import full backup data from MySQL into a TiDB cluster using TiDB Lightning;
-3. Replicate the incremental data of MySQL into TiDB.
+#### Data replication between TiDB/MySQL clusters
 
-If the volume is less than TBs of data, it is recommended to migrate MySQL data to TiDB using DM (the migrating process includes full data import and incremental data replication).
+You can use TiDB Binlog to replicate data between TiDB clusters. You can also use TiDB Binlog to replicate data to the downstream MySQL cluster.
 
-#### Replication of TiDB cluster data
+#### Full backup and restore of the data in TiDB/MySQL clusters
 
-It is recommended that you use TiDB Binlog to replicate TiDB data to downstream TiDB/MySQL.
+To restore data to a TiDB cluster:
 
-#### Full backup and restore of TiDB cluster data
+- Use the BR tool for both full data backup and full data restore
 
-The recommended steps are:
+To restore data to a MySQL cluster:
 
-1. Back up full data using Mydumper;
-2. Restore full data into TiDB/MySQL using TiDB Lightning.
+- Use the Mydumper tool for full data backup
+- Use the Lightning tool for full data restore
```

Comment:
lightning(tidb backend) is more clear

Reply:
Updated to "Use the Lightning tool with `tidb` backend for full data restore"